<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageNetworkTor</class>
 <widget class="QWizardPage" name="PageNetworkTor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>618</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>How should Feather route its network traffic?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>By default, Feather routes most traffic over Tor. </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>An exception is made for the initial wallet synchronization after opening a wallet. Synchronization requires a lot of data transfer and is therefore very slow over Tor. </string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>On Tails, Whonix, or when Feather is started with Torsocks, all traffic is routed through Tor regardless of application configuration.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Connections to local nodes are never routed over Tor.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radio_useDefaultSettings">
     <property name="text">
      <string>Use default settings (recommended)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="radio_configureManually">
     <property name="text">
      <string>Configure manually</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_privacyLevel">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="icon_allTorExceptNode">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>icon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_allTorExceptNode">
          <property name="text">
           <string>Route all traffic over Tor, except traffic to node</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">btnGroup_privacyLevel</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="icon_allTorExceptInitSync">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>icon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_allTorExceptInitSync">
          <property name="text">
           <string>Route all traffic over Tor, except initial wallet synchronization</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">btnGroup_privacyLevel</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="icon_allTor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>icon</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_allTor">
          <property name="text">
           <string>Route all traffic over Tor</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">btnGroup_privacyLevel</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="btnGroup_privacyLevel"/>
 </buttongroups>
</ui>
